src/docs/slot_specifications.md

```diff
@@ -0,0 +1,206 @@
+# (Unoffical) MIxS LinkML Slot Specifications
+
+The key words “MUST”, “MUST NOT”, “SHOULD”, etc. are to be interpreted as described in RFC 2119.
+
+| Metadata     | Value                         |
+| ------------ | ----------------------------- |
+| Version      | 0.0.1                         |
+| Last updated | 2025-05-05                    |
+| Authors      | James Fellows Yates (@jfy133) |
+
+## 1. General
+
+### 1.1 LinkML compatibility
+
+A MIxS slot MUST be written in and compatible with the [LinkML](https://linkml.io/) model, and any of it's requirements (e.g. in YAML format).
```

Comment:
who is this documentation for? The GSC have "terms" not slots, slots is a LinkML thing. If this documentation is meant for general users then we should make it sound like its GSC terms we're talking about.

Reply:
I'm not entirely comfortable with introducing the phrase "MIxS slot", a slot is definately a LinkML thing, not a MIxS thing. I would prefer either MIxS term, or just use the word slot without prefixing with MIxS.

Reply:
Removed all references to MIxS slot